backend/src/test/resources/backend_config.json

```diff
@@ -5,22 +5,24 @@
     "organisms": {
         "dummyOrganism": {
             "referenceGenomes": {
-                "nucleotideSequences": [
-                    {
-                        "name": "main",
-                        "sequence": "ATTAAAGGTTTATACCTTCCCAGGTAACAAACCAACCAACTTTCGATCT"
-                    }
-                ],
-                "genes": [
-                    {
-                        "name": "someLongGene",
-                        "sequence": "AAAAAAAAAAAAAAAAAAAAAAAAA"
-                    },
-                    {
-                        "name": "someShortGene",
-                        "sequence": "MADS"
-                    }
-                ]
+                "dummyOrganism": {
```

Comment:
To me, it is quite confusing to have the same name here as the organism (also applies to the configs below), raising questions whether this needs to be the same and is somehow used for linking things (which are not the case). If we don't like `main`, can we find another term? E.g., `theOnlyReferenceGenome`, `referenceGenome1`, `mainReferenceGenome`?

Reply:
The key doesn't really matter for now. I don't know yet whether it will matter, but I could imagine that the logic becomes simpler if e.g. "is single pathogen" (which we will need in some places I assume) is equivalent to `"mainReferenceGenome" in referenceGenomes`.

Should I change them to `mainReferenceGenome`?
* changing the exact name later should be relatively easy
* if we notice that the key does no matter we can still silently remove the restriction. Going from specific to generic is not a breaking change (compared to the other way round when we suddenly need to enforce a specific name in those cases)

Reply:
Agree it doesn't matter much. I think for a single name my preference would be `singleReference`. `main` to me implies the existence of something else that is not main.

Reply:
`singleReference` sounds great to me!

Reply:
I feel like it's better to call what we're doing "allowing subOrganisms" rather than "multiple reference genomes". As a result I think we should use `singleSuborganism`

Reply:
As discussed: We stick with `singleReference` for now.